recognition/scoring.py

```diff
@@ -149,31 +156,61 @@ def run(self, output_to_report_dir=True):
 
         if output_to_report_dir:  # run as real job
             with open(f"{output_dir}/sclite.dtl", "rt", errors="ignore") as f:
+                # Example:
+                """
+                Percent Total Error       =    5.3%   (2709)
+                ...
+                Percent Word Accuracy     =   94.7%
+                ...
+                Ref. words                =           (50948)
+                """
+
+                # key -> percentage, absolute
+                output_variables: Dict[str, Tuple[Optional[tk.Variable], Optional[tk.Variable]]] = {
+                    "Percent Total Error": (self.out_wer, self.out_num_errors),
+                    "Percent Correct": (self.out_percent_correct, self.out_num_correct),
+                    "Percent Substitution": (self.out_percent_substitution, self.out_num_substitution),
+                    "Percent Deletions": (self.out_percent_deletions, self.out_num_deletions),
+                    "Percent Insertions": (self.out_percent_insertions, self.out_num_insertions),
+                    "Percent Word Accuracy": (self.out_percent_word_accuracy, None),
+                    "Ref. words": (None, self.out_ref_words),
+                    "Hyp. words": (None, self.out_hyp_words),
+                    "Aligned words": (None, self.out_aligned_words),
+                }
+
+                outputs_absolute: Dict[str, int] = {}
                 for line in f:
```

Comment:
Right now, we always go through all lines, right? We might introduce an early stop (e.g. if the line starts with `"CONFUSION PAIRS"`) because that can be thousands of lines while we are only interested in the first 30 or so.

Reply:
Yes. I did not change this logic, but I can change that. `Aligned words` should be the last entry.